test/libsolidity/SolidityExecutionFramework.cpp

```diff
@@ -85,3 +109,15 @@ bytes SolidityExecutionFramework::compileContract(
 		cout << "metadata: " << m_compiler.metadata(contractName) << endl;
 	return obj.bytecode;
 }
+
+void SolidityExecutionFramework::addPreamble(string& _sourceCode)
```

Comment:
Parameters used for input and output are supposed to have `io_` prefix (rather than `_`):
```suggestion
void SolidityExecutionFramework::addPreamble(string& io_sourceCode)
```

Also, I think it would be better if this function returned a string instead of modifying its parameter. We can always use `move()` if we don't actually want a copy.

Reply:
Switched to `move()` ing modified input as output